src/libraries/System.Runtime.InteropServices.JavaScript/src/ILLinkTrim.xml

```diff
@@ -0,0 +1,5 @@
+﻿<linker>
+    <assembly fullname="System.Runtime.InteropServices.JavaScript">
+        <type fullname="System.Runtime.InteropServices.JavaScript.Runtime" />
```

Comment:
@marek-safar @vitek-karas - this seems like a good place to add a feature switch so we aren’t unconditionally rooting this type all the time.

Reply:
In general I agree - I'm just not sure when we would use the feature switch. For Blazor it needs to be on, otherwise nothing works (right?). Do we have any other target which uses this assembly (where it would make sense to turn it off)?

Reply:
No feature switch please. There is nothing user/scenario controllable

Reply:
OK, if there is never a use case for this assembly outside of the current browser-wasm usages, then rooting this type unconditionally seems reasonable.